wire-ios-utilities/Source/NSManagedObjectContext+PerformGrouped.swift

```diff
@@ -59,6 +59,9 @@ public extension NSManagedObjectContext {
         }
 
         if let error = thrownError {
+            groups.apply {
+                dispatchGroupContext?.leave($0)
+            }
```

Comment:
opinion: we can avoid those syntax sugar, that doesn't make things simpler or better understandable.
Is this custom func `apply` easier to understand than?
```swift
if let groups {
    dispatchGroupContext?.leave(groups)
}
```

Reply:
Aren't you missing a for loop? @markusfassbender

Reply:
No, the leave function takes an array and loops inside.

Reply:
better to remove the sugar from project in another PR